scripts/installation/install.sh

```diff
@@ -77,7 +81,89 @@ delete_crds() {
   kubectl delete -f build/crds --timeout=90s
 }
 
+create_kb(){
+  cd ${SEDNA_ROOT}
+  kubectl label node/$KB_NODE_NAME sedna=controller --overwrite
+
+  kubectl $action -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: kb
+  namespace: sedna
+spec:
+  selector:
+    sedna: controller
+  type: NodePort
+  ports:
+    - protocol: TCP
+      port: 9020
+      targetPort: 9020
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kb
+  labels:
+    sedna: controller
+  namespace: sedna
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      sedna: controller
+  template:
+    metadata:
+      labels:
+        sedna: controller
+    spec:
+      nodeSelector:
+        sedna: controller
+      serviceAccountName: sedna
+      containers:
+      - name: kb
+        imagePullPolicy: IfNotPresent
+        image: kubeedge/sedna-kb:v0.1.0
+        command: ["python", "/code/lib/sedna/service/run_kb.py"]
+        env:
+          - name: LIFELONG_KB_URL
+            value: "sqlite:///db/lifelong_kb.sqlite3"
+        volumeMounts:
+        - name: lifelong-kb-url
+          mountPath: /db
+        resources:
+          requests:
+            memory: 256Mi
+            cpu: 512m
```

Comment:
request for cpu is 512m, which is too large, since it's a server, not high-cpu-intensity batch job. 
And unit of cpu is decimal

Reply:
100m is recommended